content/ui/enforce-onboarding-quickstart.md

```diff
@@ -243,4 +240,13 @@ chainctl cluster uninstall
 kind delete cluster --name enforce-demo
 ```
 
-To learn more about Chainguard Enforce, please review our documentation and other resources on Chainguard Academy.
+## Learn more
```

Comment:
@amdawson — I added a number of links to the UI doc for learning more about Chainguard Enforce. Let me know if there are others we should highlight right now. I think once I know what changes you want to the arch diagram we can add that link too